src/types/index.ts

```diff
@@ -0,0 +1 @@
+export * from './buoy.type';
```

Comment:
What is the purpose of this file?

Reply:
This file enables us to write `import { ParsedBuoy } from 'src/types';` instead of `import { ParsedBuoy } from 'src/types/buoy.type';`.

When you do not specify a file when importing from a folder, the compiler will look for the index file. When present that file is implicitly considered to be that folder’s “main” module.<sup>1</sup>

##### (1) The full Node.js resolution algorithm is outlined [here](https://nodejs.org/api/modules.html#modules_all_together), if you're curious.